css/style.css

```diff
@@ -5,6 +5,81 @@ body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
+ul, li{
+    list-style-type: none;
+    
+}
+.navigation {
+    background-color: #fff;
+    
+}
+.navigation .links {
+    float: right;
+    color: #838994;
+    font-size: 1.04rem;
+    margin-top: 1.7rem;
+}
+.navigation .links li {
+    display: inline;
+    margin-left: 1.563rem;
```

Comment:
How does the navigation bar look towards the right? There doesn't seem to be any space between "login" and the edge of the screen, maybe using flex would be good here to help evenly space the elements.